lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -569,6 +569,37 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return true;
 	});
 
+	/**
+	* Tests whether at least one element in the typed array passes the test implemented by the provided function.
+	*
+	* @name some
+	* @memberof TypedArray.prototype
+	* @type {Function}
+	* @param {Function} fcn - predicate function
```

Comment:
```suggestion
	* @param {Function} predicate - predicate function
```

I suggest renaming here and below in your implementation.